src/main/java/com/garbagemule/MobArena/formula/FormulaManager.java

```diff
@@ -20,32 +20,26 @@ public class FormulaManager {
         this.parser = parser;
     }
 
-    @SuppressWarnings("unused")
```

Comment:
In IntelliJ, the `registerConstant()`, `registerUnaryOperator()`, and `registerBinaryOperator()` methods are all marked as "unused", which is why these suppressions are here.

Turns out it doesn't produce warnings for the other methods, because these are used in the test suite. I think I will try to buff out the test suite a bit to include tests of these methods as well in order to back up the removal of the suppress annotations.